src/app/modules/appointment/edit-appointment/edit-appointment.component.html

```diff
@@ -39,18 +39,18 @@
                       id="endtime"
                       [showTime]="true"
                       hourFormat="24"
-                      placeholder="End Time"
+                      placeholder="{{ 'editappointments.END' | translate }}"
                     ></p-calendar>
                   </div>
                 </div>
               </div>
               <div class="p-col-12 p-md-6 p-lg-4">
                 <div class="p-col-12">
                   <div class="p-field p-fluid">
-                    <label for="status">Status *</label>
+                    <label for="status">{{ 'editappointments.STATUS' | translate }}</label>
```

Comment:
Hast du die Pflichtfeld-Sternchen absichtlich weggelassen?

Reply:
oups nein, ein versehen.. überall sonst müsste ich es aber gemacht haben.

Reply:
Ich denke, wir sollten die Sternchen als dreingäbe sehen und daher sollten sie nicht Teil der Übersetzung sein. Es kann ja auch ein Status Label ohne Sternchen geben. Dadurch müsstest du jetzt für diese beiden Zustände eine Übersetzung anlegen.

Reply:
Nicht in die Übersetzung reinnehmen, sondern hinter die }} Klammern setzen